.github/workflows/ci.yml

```diff
@@ -105,27 +101,12 @@ jobs:
       - name: Test
         run: bash ./ci.sh
 
-      # Don't cache "master" toolchain, it's a waste.
       - name: Before caching
         run: |
-          # Binary artifacts is saved in `ci/bin` dir.
+          # Strip files in `ci/bin` dir to remove useless debug-info.
           strip ~/.cargo/ci/bin/*
```

Comment:
Can you just add this to the `Install RTIM and xargo` step? That seems the easiest to do and also is easier to understand conceptually.